documentation/staging/content/userguide/managing-operators/installation.md

```diff
@@ -155,6 +157,9 @@ see [Troubleshooting]({{<relref "/userguide/managing-operators/troubleshooting#c
   A backslash (`\`) is only required when specifying the selector on the command line using `--set`,
   as shown in the previous example.
```

Comment:
Should the fact that the webhook is implicitly installed be mentioned in these notes (just like in the upgrade section)? Or the potential need to use the 'only' uption or 'do not implicitly uninstall' option?  

Alternatively, combine discussion about WH into one location and point to this location both from these notes and from the upgrade section.

Reply:
Changed to mention the fact the webhook is implicitly installed and added reference to `installing the coversion webhook` section of the WH document.